reliability-scanner/Makefile

```diff
@@ -1,6 +1,6 @@
 REGISTRY = "projects.registry.vmware.com/cre"
 IMAGE = "reliability-scanner"
-TAG = "v0.1.1"
+TAG = "v0.1.2"
```

Comment:
Maybe these could be optional to allow someone to change them at build time? `REGISTRY="my-cool-repo" make run`

``` sh
REGISTRY ?= "projects.registry.vmware.com/cre"
IMAGE ?= "reliability-scanner"
TAG ?= "v0.1.2"
```